deploy/operator.yaml

```diff
@@ -2462,3 +2505,5 @@ spec:
               value: "k8s.gcr.io/sig-storage/csi-provisioner:v2.2.1"
             - name: VERBOSITY
               value: "3"
+            - name: MONITORING_NAMESPACE
+              value: ""
```

Comment:
Should this be blank? Would it be possible to use the WATCH_NAMESPACE instead?

Reply:
Yeah, it should be blank here (upstream) so we default to "monitoring" in `getMonitoringNamespace()`
(downstream it's openshift-monitoring).

We cant use the `WATCH_NAMESPACE` as it's not the one where the user installed their Prometheus resources